rs/nns/governance/README.md

```diff
@@ -0,0 +1,9 @@
+The (NNS) governance canister is the "heart" of the Network Nervous System (NNS). This is where
+people stake ICP (i.e. make it not liquid) in order to create "neurons". Using their neurons, they
+can then make proposals and vote on them. Proposals can do things like
+
+  1. Set the code that operates the Internet Computer (IC), including the governance canister itself.
```

Comment:
```suggestion
  1. Set the code that operates the Internet Computer (IC), including the Governance canister itself.
```

Reply:
I think we usually do not capitalize. However, we are not consistent on this. FWIW, I found [231 occurrences of "Governance"](https://sourcegraph.com/search?q=context:%40daniel.wong1/ic+%5E%5Cs%2B//+%5CbGovernance%5Cb&patternType=regexp&case=yes&sm=1&groupBy=repo) in comments and [444 occurrences of "governance"](https://sourcegraph.com/search?q=context:%40daniel.wong1/ic+%5E%5Cs%2B//+%5Cbgovernance%5Cb&patternType=regexp&case=yes&sm=1&groupBy=repo).